FC-0068: Docs Cleanup (#649)
openedx/docs.openedx.org: source/educators/how-tos/set_up_course/edit_course_about.rst

Change the Course Title
=======================

To change the way that your course title appears in Studio and the LMS, follow these steps.

.. note::

>  EdX does not recommended that you do this if your course is live.

Reviewer: ```suggestion
 It is not recommended that you do this if your course is live.
```

Did your search also search for instances of the string "EdX"?

Author: I think it was a manual read through each doc. I'll do a batch search.
